content/01-install/02c-install-wallet-Mac.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Install Wallet on Apple Mac
+metaTitle: Install Wallet on Apple Mac
+---
+
+>Author's note to reviewers: The original document describes disk space requirements for syncing and mining. I didn't think a wallet did those things. Please clarify. Also please check the Mac instructions; I do not have a Mac machine. 
+
+There are two Mantis applications you can use: the Mantis client and the Mantis wallet. You can download either or both.
+
+This section outlines how to install the Mantis wallet on your Apple Mac machine. For instructions to install the Mantis client, you can go [here](/install/install-client-mac).
+
+## Prerequisites
+
+These are the prerequisites for installing Mantis Wallet:
+* A Java Virtual Machine (JVM) with version 1.8.x.
+    To check the JVM version, use `java -version`.
+    To install Java, follow [these instructions](https://java.com/en/download/help/download_options.html).
+    (Note that  the Mantis team has not tested the wallet with JVM 1.9 or later versions)
+* 4G of memory (RAM)
+* 250GB of disk space.
+
+##  Installing the wallet
+
+To install Mantis wallet, follow these steps:
+
+1. Choose and download the Apple Mac installer from the wallet quick-start page:
+![Apple Mac Installer](../images/2c-Download-Installer.png)
+Note the checksum.
+```
+Mantis-Wallet-<version>.AppImage
+Mantis-Wallet-<version>.dmg
+Mantis-Wallet.Setup.<version>.exe
+```
+> Remember to run checksum verification on your downloads. Refer to [this section](first-steps/downloads#checksum) for instructions.
+
+4. Move the binary file to your preferred folder and unzip the client archive file. Finally, remove the zip file.
+
+```
+mv Mantis-Wallet-3.1.0.AppImage /home/<user>/bin/
```

Comment:
on Mac OS you just mount the .dmg file and proceed with the typical mac os installation procedure